TF-440: Terraform Enterprise Demo Installation Mode Removal (#2234)
hashicorp/terraform-website: content/enterprise/before-installing/data-storage-requirements/postgres-requirements.mdx

When providing optional extra keyword parameters for the database connection,
note an additional restriction on the `sslmode` parameter is that only the

> `require`, `verify-full`, `verify-ca`, and `disable` values are allowed. The default value of `sslmode` is set to `require` for _External Services_ installations or `disable` for _Mounted Disk_ installations.

Reviewer: Should it be and? 

```suggestion
`require`, `verify-full`, `verify-ca`, and `disable` values are allowed. The default value of `sslmode` is set to `require` for _External Services_ installations and `disable` for _Mounted Disk_ installations.
```

When providing optional extra keyword parameters for the database connection,
note an additional restriction on the `sslmode` parameter is that only the

> `require`, `verify-full`, `verify-ca`, and `disable` values are allowed. The
> default value of `sslmode` is set to `require` for _External Services_
> installations and `disable` for _Mounted Disk_ installations.

Reviewer: ```suggestion
`require`, `verify-full`, `verify-ca`, and `disable` values are allowed. For installations in External Services mode , the default value of `sslmode` is set to `require`. For installations in Mounted Disk mode, the default value of `sslmode` is set to `disable`.
```

-> **Note:** See the PostgreSQL library documentation for more about [extra parameters related to sslmode](https://www.postgresql.org/docs/12/libpq-ssl.html). Terraform Enterprise provides a certificates file at `/tmp/cust-ca-certificates.crt`, which is required by the `verify-full` and `verify-ca` modes. Additional certificates can be added via the [CA Custom Bundle](/enterprise/install/interactive/installer#certificate-authority-ca-bundle) setting.

-> **Note:** The [Client Certificates](https://www.postgresql.org/docs/12/libpq-ssl.html) configuration is currently not supported by Terraform Enterprise due to the limitation of storing certificate files for the `sslcert`, and `sslkey` connection parameters.

## PostgreSQL 9.5 to 12 Upgrade

> In Terraform Enterprise v202103-1, the internally-managed PostgreSQL server has

Reviewer 2: ```suggestion
In Terraform Enterprise v202103-1, the internally-managed PostgreSQL server was
```

Just a nit-pick, this should be in past tense now, I think.

> been upgraded from PostgreSQL 9.5 to PostgreSQL 12. This change only affects

Reviewer 2: ```suggestion
upgraded from PostgreSQL 9.5 to PostgreSQL 12. This change only affected
```

> In Terraform Enterprise v202103-1, the internally-managed PostgreSQL server was
> been upgraded from PostgreSQL 9.5 to PostgreSQL 12. This change only affected
> _Mounted Disk_ installations.

Reviewer: ```suggestion
In Terraform Enterprise v202103-1, the internally-managed PostgreSQL server was
been upgraded from PostgreSQL 9.5 to PostgreSQL 12. This change only affected
installations in Mounted Disk mode.
```
